### ISSUE TITLE
Missing page "Semihosting for AArch32 and AArch64"

### ISSUE THREAD

**Issue by harrisonmutai-arm** [ARM-software/software-standards#226]
References to the semihosting documentation appear to be broken:

https://developer.arm.com/documentation/100863/latest

This redirects to the GitHub release page, from [this](https://developer.arm.com/documentation/100961/1122/Programming-Reference/Semihosting?lang=en) page. u-boot points to this page as well.

**Comment by stuij** [comment]
to be clear: the latest releases of the ABI and related documentation are now exclusively hosted on Github. I'm not sure where those links to developer.arm.com are coming from, but I did a grep through the ABI documentation, and I didn't find a reference in there at least.

**Comment by harrisonmutai-arm** [comment]
Apologies @stuij, I've only just seen your response! 
> See the Semihosting doc link under Miscellaneous material, as it's better I think to link to the latest released documentation, instead of to the latest release from head.
This make sense to me. Although the issue I raised relates more to links from developer.arm.com. The page with the broken link I was originally referring to is here: https://developer.arm.com/documentation/100961/1122/Programming-Reference/Semihosting?lang=en. My general expectation would be that the links here would redirect to a render of the page rather than the source. Perhaps this isn't the right place to raise issues relating to developer.arm.com?
![image](https://github.com/ARM-software/abi-aa/assets/97464547/8be798f4-476a-4000-8b92-d26d64b7772c)

**Comment by stuij** [comment]
Yes, we who deal with the docs here are unrelated to the developer.arm.com maintainers. Best to take this up with them directly, but I do know that a number of people would prefer or at least would like the option to use the PDF version. If you link straight to the web version, it'll be non-obvious to them where to find it. At least this way you have the option at the expense of extra clicking.
